### ISSUE TITLE
Social media navigation only in Left header Layout

### ISSUE THREAD

**Issue by rolikeusch** [Manoz/split-me#4]
It looks like the Social media navigation is only visible in the Left header Layout, if i use the Top header Layout i can't see the Social Media Icons.
Or do i something wrong?

![left header](https://cloud.githubusercontent.com/assets/3592156/4933461/6e9eded6-6597-11e4-89d8-2ec760aa7cc7.png)
![top header](https://cloud.githubusercontent.com/assets/3592156/4933462/6ebee604-6597-11e4-9de6-da3b6a8cf7fb.png)

**Comment by Manoz** [comment]
My bad :) I'll fix it. Thank you

**Comment by rolikeusch** [comment]
Uff great to hear it's not my fault ;)
Any Plans to implement a Header Logo instead of Header Text in the Theme Options?

**Comment by Manoz** [comment]
Hum I don't know. Split Me was my very first theme. Now I'm more skilled on WordPress and I'm trying to fix lot of old things but I also wanted to build a way better version from scratch of Split Me. (Démo here http://alubidium.mwanoz.fr/ ). I'll try to push an update as soon as possible :)

**Comment by rolikeusch** [comment]
Oh Alubidium looks also nice, almost to much Feature for me ;) But this is not yet available or?

**Comment by Manoz** [comment]
Nope, it's just a beta, the theme isn't finished yet. Lot of people asks me to add some widgets in Split Me but I didn't want to add a widget area on Split Me. I wanted a very light theme. That's why I'm building another 2 column theme :)

And I like the new comments area :p 

![screenshot_2014-11-06_21-01-09](https://cloud.githubusercontent.com/assets/1554207/4942924/f588c436-65ef-11e4-9ae1-61ee7292fff9.png)

**Comment by rolikeusch** [comment]
I like that fact that Split Me is very "lightweight", i don't need widgets... just the Social Media Icons ;)
The new comments area looks nice!

**Comment by Manoz** [comment]
Yep. I'll try to push the update today :)

**Comment by rolikeusch** [comment]
There's no hurry.

An other Problem that i have, if i choose not to show the Header Text in the Theme settings, the Header in mobile view is complete gone.
